pages/latest/[urlTitle].tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+
+// Graphql
+import { NAVIGATION_MENU_QUERY, FULL_PAGE_QUERY } from '../../graphql/queries';
```

Comment:
Add `graphql ` to the `tsconfig` like this:

https://github.com/dotCMS/documentation/blob/main/tsconfig.json#L19